release-notes/opensearch-sql.release-notes-2.6.0.0.md

```diff
@@ -0,0 +1,42 @@
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.6.0
+
+### Enhancements
+
+* Extend comparison methods to accept different datetime types ([#129](https://github.com/opensearch-project/sql/pull/129) [#1196](https://github.com/opensearch-project/sql/pull/1196))
+* Enable concat() string function to support multiple string arguments ([#1279](https://github.com/opensearch-project/sql/pull/1279))
+* Add more keywords as identifier in PPL ([#1319](https://github.com/opensearch-project/sql/pull/1319))
+* Update DATE_ADD/ADDDATE and DATE_SUB/SUBDATE functions ([#112](https://github.com/opensearch-project/sql/issues/122) [#1182](https://github.com/opensearch-project/sql/pull/1182))
+* Escape character support for string literals ([#1206](https://github.com/opensearch-project/sql/pull/1206))
+* Updated EXPM1() and Tests to New Engine ([#1334](https://github.com/opensearch-project/sql/pull/1334))
+* Update TIMESTAMP function implementation and signatures ([#1254](https://github.com/opensearch-project/sql/pull/1254))
+* Add GET_FORMAT Function To OpenSearch SQL Plugin ([#1299](https://github.com/opensearch-project/sql/pull/1299))
+* Add TIME_FORMAT() Function To SQL Plugin ([#1301](https://github.com/opensearch-project/sql/pull/1301))
+* Support More Formats For GET_FORMAT Function ([#1343](https://github.com/opensearch-project/sql/pull/1343))
+* Add last_day Function To OpenSearch SQL Plugin ([#1344](https://github.com/opensearch-project/sql/pull/1344))
+* Add WeekOfYear Function To OpenSearch ([#1345](https://github.com/opensearch-project/sql/pull/1345))
+
+### Bug Fixes
+
+* Allow literal in aggregation ([#203](https://github.com/opensearch-project/sql/issues/203) [#1288](https://github.com/opensearch-project/sql/pull/1288))
```

Comment:
Maybe: remove these links? 203 - is a PR on BQ fork actually, so link to 203 on sql repo may be confusing.
```suggestion
* Allow literal in aggregation [#1288](https://github.com/opensearch-project/sql/pull/1288))
```

Reply:
There are similar links in other places too

Reply:
Thanks for noticing this. Removed all misleading links.